packages/cactus-plugin-ledger-connector-iroha/src/test/typescript/unit/iroha-test-ledger-parameters.test.ts

```diff
@@ -0,0 +1,36 @@
+// import test, { Test } from "tape";
```

Comment:
everything in the file is commented

Reply:
This file is probably not needed anymore, I am going to ask Peter today about this file.
The functionality of this file has been moved to another file in the other PR. 
(https://github.com/hyperledger/cactus/blob/dc0199e4b5ae384be1b04ad73e5ef67f4efc8f0e/packages/cactus-test-tooling/src/test/typescript/integration/iroha/iroha-test-ledger/constructor-validates-options.test.ts)
Right now the file is commented out so that the code could be compiled.

Reply:
Sorry I talked with Peter yesterday, and we decided that we still want that. Will fix the problem!

Reply:
Fixed!